### ISSUE TITLE
H6110 Light Strip Not Able To Control

### ISSUE THREAD

**Issue by nackerr** [bwp91/homebridge-govee#410]
### What issue do you have? Please be as thorough and explicit as possible.

Can't control H6110 light strip. Device shows up, and Home app shows the model number and version, but no controls work.

### Details of your setup.
* Do you use (1) Homebridge UI-X (2) Homebridge CLI or (3) HOOBS? 

Homebridge UI-X

* Which version of Homebridge/HOOBS do you have?

Homebridge 1.6.0

* Which platform do you run Homebridge/HOOBS on (e.g. Raspberry Pi/Windows/HOOBS Box)? Please also mention your version of Node.js/NPM if known.

Linux, Ubuntu 22.04
NodeJS 18.12.1

* Which version of this plugin (homebridge-govee) do you have? Has the issue started since upgrading from a previous version?

homebridge-govee v7.7.2-beta.7

* Which Govee devices do you have that are causing issues? Please include product models if applicable.

H6110

### Please paste any relevant logs below.

```
[1/3/2023, 12:33:38 AM] [Govee] [Gaming Room Strip] [AWS] sending update {"msg":{"cmd":"turn","cmdVersion":0,"data":{"val":1},"transaction":"v_1672724018685000","type":1}}.
[1/3/2023, 12:33:38 AM] [Govee] [Gaming Room Strip] current state [on].
[1/3/2023, 12:33:39 AM] [Govee] [AWS] message event [{"proType":0,"msg":"{\"transaction\":\"v_1672724018685000\",\"sku\":\"H6110\",\"device\":\"9E:12:A4:C1:38:BF:B4:C6\",\"cmdVersion\":0,\"type\":0,\"cmd\":\"status\",\"data\":\"{\\\"softversion\\\":\\\"1.07.02\\\",\\\"wifiSoftVersion\\\":\\\"1.03.05\\\",\\\"turn\\\":1,\\\"brightness\\\":254,\\\"modeValue\\\":[\\\"qgUC////Af+0awAAAAAAAAAAAHM=\\\"],\\\"mode\\\":2,\\\"timer\\\":{\\\"enable\\\":0,\\\"time\\\":[{\\\"openHour\\\":0,\\\"openMin\\\":0,\\\"closeHour\\\":23,\\\"closeMin\\\":59}]},\\\"color\\\":{\\\"red\\\":255,\\\"green\\\":180,\\\"blue\\\":107},\\\"colorTemInKelvin\\\":3000}\"}","state":{"onOff":1,"brightness":254,"color":{"r":255,"g":180,"b":107},"colorTemInKelvin":3000,"connected":"true","sku":"H6110","device":"9E:12:A4:C1:38:BF:B4:C6"}}].
```

As you can see it tries to work, but the actions aren't sent to the Lightstrip or to the Govee app.

**Comment by radd101** [comment]
Same issue here. 
I've reinstalled the plugin, tried the plugin betas, tried using child bridges, tried a separate wifi network, requested new API key. Logs show similar results as OPs. 
All functions are fine in the Govee app. 
Now that I think of it, I did that HomeKit architecture update thing. Wonder if that has something to do with it. Pretty sure I would have remembered if the upgrade did it, these strips are used all the time.

**Comment by jnpatrick99** [comment]
Same here! Started to happen yesterday. The button in UI shows it's on but in reality it's not.

**Comment by jnpatrick99** [comment]
Installed the latest version of everything and enabled debug.

Now I have an error that openssl is not found. How do I fix it? I'm on Windows 10.

UPDATE: Installed latest version of OpenSSL 3.0.7, added it to Path and restarted - still not working.

UPDATE2: So apparently I had to set OPENSSL_BIN system OS variable pointing to openssl.exe. The AWS initialized but it's still not working! The error is ``openssl pkcs12 -in C:\WINDOWS\TEMP\389197a5171607d4af66ced431c89742a22b72f3 -passin file:C:\WINDOWS\TEMP\43a77c81983cc078deb081d96d4878af7b34356c -legacy -traditional -nodes
pkcs12: unable to load provider legacy``

UPDATE3: I got rid of the errors and it's partially working! I had to define another OS variable called OPENSSL_MODULES pointing to the bin directory, the same place where openssl.exe was located.
1) It only works with 1 device (I have 2 on my network, both H6110).
2) It doesn't correctly display its status - when it's ON it says OFF
